.dockerignore

```diff
@@ -0,0 +1,4 @@
+.env
+.env.template
+LICENSE
+README.md
```

Comment:
We can add the following files,

- AUTHORS
- Dockerfile
- Pipfile
- Pipfile.lock
- Procfile
- runtime.txt